src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs

```diff
@@ -112,8 +113,9 @@ public void DnsObsoleteGetHostByName_EmptyString_ReturnsHostName()
             Assert.Contains(Dns.GetHostName(), entry.HostName, StringComparison.OrdinalIgnoreCase);
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/1488", TestPlatforms.OSX)]
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process), nameof(PlatformDetection.IsThreadingSupported))] // [ActiveIssue("https://github.com/dotnet/runtime/issues/27622")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/1488", TestPlatforms.OSX)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55271", TestPlatforms.Linux)]
```

Comment:
I think the DNS tests are environmental. WE should disable them only on failing distro.

Reply:
I don't see a way to disable in SLED.

Reply:
I think I see how.